tests/test_spdx_matcher.py

```diff
@@ -79,7 +82,7 @@ def test_backtracking_challenging(self):
 
         analysis, match = spdx_matcher.analyse_license_text(CHALLENGING)
 
-        self.assertEqual(len(analysis["licenses"]), 24)
+        self.assertEqual(len(analysis["licenses"]), 16)
```

Comment:
some match result is deprecated, including:
`GPL-2.0`, `GPL-1.0+`, `GPL-1.0`, `LGPL-2.0+`, `LGPL-2.1`, `GPL-3.0`, `GPL-3.0+`, `LGPL-2.1+`, `LGPL-2.0`